tape/src/test/java/com/squareup/tape2/QueueFileTest.java

```diff
@@ -695,15 +694,27 @@ private QueueFile newQueueFile(boolean zero) throws IOException {
     byte[] data = new byte[firstElementPadding];
     queue.raf.seek(headerLength);
     queue.raf.readFully(data, 0, firstElementPadding);
-    assertThat(data).containsOnly((byte) 0x00);
+    assertThat(containsOnly(data, (byte) 0x00)).isTrue();
 
     // Read from the last element to the end and make sure it's zeroed.
     int endOfLastElement = headerLength + firstElementPadding + 4 * (Element.HEADER_LENGTH + 1024);
     int readLength = (int) (queue.raf.length() - endOfLastElement);
     data = new byte[readLength];
     queue.raf.seek(endOfLastElement);
     queue.raf.readFully(data, 0, readLength);
-    assertThat(data).containsOnly((byte) 0x00);
+    assertThat(containsOnly(data, (byte) 0x00)).isTrue();
+  }
+
+  private static boolean containsOnly(byte[] arr, byte element) {
```

Comment:
uh, is there a way to do this with Truth? With Collections, you can do `containsExactly`, but that's not what we want. Should I file an issue or PR for Truth for `containsOnly`?